base/applications/magnify/magnifier.c

```diff
@@ -923,9 +924,15 @@ INT_PTR CALLBACK OptionsProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lPar
                 if (HIWORD(wParam) == CBN_SELCHANGE)
                 {
                     HWND hCombo = GetDlgItem(hDlg,IDC_ZOOM);
+					LPCTSTR currentZoomValue = "";
```

Comment:
Please fix the indentation by remaining consistent with the rest of the file: no tabs, 4-spaces.